specs/protocol/derivation.md

```diff
@@ -199,6 +199,7 @@ protocol upgrades.
 | `max_sequencer_drift` | 600 | 1800 | [Fjord](../fjord/derivation.md#constant-maximum-sequencer-drift) | Changed from a chain parameter to a constant with Fjord. |
 | `MAX_RLP_BYTES_PER_CHANNEL` | 10,000,000 | 100,000,000 | [Fjord](../fjord/derivation.md#increasing-max_rlp_bytes_per_channel-and-max_channel_bank_size) | Constant increased with Fjord. |
 | `MAX_CHANNEL_BANK_SIZE` | 100,000,000 | 1,000,000,000 | [Fjord](../fjord/derivation.md#increasing-max_rlp_bytes_per_channel-and-max_channel_bank_size) | Constant increased with Fjord. |
+| `MAX_SPAN_BATCH_ELEMENT_COUNT` | 10,000,000 | 10,000,000 | | Number of elements |
```

Comment:
Maybe also say something like "Effectively introduced with Fjord" and link to the `Span Batch Size Limits` section? It may also make sense to add one sentence to the Fjord specs section about increasing the `MAX_RLP_BYTES_PER_CHANNEL` constant, that this new constant takes effect with it. This may make it easier to implement the Fjord diff in existing implementations.